docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes.md

```diff
@@ -91,9 +91,9 @@ For any third-party Cron scheduling tools, you can disable Crontab and add the f
 Disabling Crontab creates alerts after the universe is created, but they can be ignored. You need to ensure Cron jobs are set appropriately for YBA to function as expected.
   {{< /tip >}}
 
-- Verify that Python 3.6 or later is installed.
+- Verify that Python 3.5-3.8 is installed. v3.6 is recommended.
 
-    If you are using Python v3.11 or later, install the selinux python package as follows:
+    If you are using Python later than v3.6, install the selinux python package as follows:
```

Comment:
They should install https://pypi.org/project/selinux/  for their corresponding python version. For example, the 3.11 command is shown, but an equivalent could be for whatever version of python they are using. If we put it after they symlink python3 the command is pretty simple python3 -m pip install selinux. We can note here that they might also need pip, which they should configure themselves. https://pip.pypa.io/en/stable/installation/ for details.

Reply:
The RedHat bug for this is https://access.redhat.com/solutions/5674911 which is why we need libselinux-python bindings on higher versions of python